index.html

```diff
@@ -219,6 +219,119 @@ <h3>Skills</h3>
 				</div>
 			</div>
 		</section>
+		<footer id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							placeholder="Full name"
+							required
+						/>
```

Comment:
- According to the project requirements, this `input` should have a `maxlength="30"` attribute. Please fix this 👍